hackathon_site/dashboard/frontend/src/slices/hardware/cartSlice.test.ts

```diff
@@ -81,3 +82,22 @@ describe("addToCart action", () => {
         });
     });
 });
+
+describe("removeFromCart action", () => {
+    test("removeFromCart removes existing item", async () => {
+        const store = makeStore();
+
+        store.dispatch(addToCart({ hardware_id: 2, quantity: 1 }));
```

Comment:
You can use make store with entities function so u don't need to dispatch this action